examples/cuda_pytorch_inference_gpt2_2gpu.yaml

```diff
@@ -0,0 +1,19 @@
+defaults:
+  - base_config # inherits from base config
+  - _self_ # for hydra 1.1 compatibility
+
+experiment_name: cuda_pytorch_inference_gpt2_2gpu
+
+model: gpt2
+device: cuda
+
+backend:
+  device_map: auto
+
+hydra:
+  job:
+    env_set:
+      CUDA_VISIBLE_DEVICES: 0,1
+  sweeper:
+    params:
+      benchmark.input_shapes.batch_size: 1,2,4
```

Comment:
for now sweeps are not tested (needs the `-m` flag in the test script) so this config will only test the default `batch_size` value.
is it necessary to test multiple batch sizes ?

Reply:
Let's keep it like that, it's fine if only the first is tested.

I added several because there was an error only when using sweeps before (PID already using the device due to the cuda context not cleaned).